### PR TITLE
Messenger bag fixes and civilian job tweaks

### LINKED ISSUE #13857: Leather satchel option doesn't work on skeleton
It spawns you in with a normal backpack. Not sure how it acts with jobs that aren't assistant, I haven't had time to test it on the main server, the test server is down, and I'm too lazy to host a local game.

Not that it really matters all that much, I think we have a total of like 4 or 5 regular skeleton players.

### LINKED ISSUE #13929: Mime backpack is all kinds of wrong
(WEB REPORT BY: diavil REMOTE: 172.93.109.202:7777)
Revision: 1c87cd4b41d49edc6ae5fa2682429fe1fb6a5ac9 

Mime starts with a goddamn industrial backpack.

#13926

Possibly the reason.

### PR DESCRIPTION
**Pull request by jknpj**
Fixes chaplain and trader jobs not spawning with messenger bags and adds plastic bags for assistants and some civilian jobs. Adds messenger bags to personal closets. 

About plastic bags:
bag

Fixes #13929 and Closes #13857

:cl:
 * bugfix: Fixes chaplain and traders not spawning with messenger bags.
 * bugfix: Fixes some civilian jobs starting with engineer bags.
 * rsadd: Adds plastic bags to assistant, bartender, chef, botanist, cargo tech, janitor and librarian starting gear.
 * rsadd: Adds messenger bags to personal closets.

### CODE REVIEW

File: code/game/jobs/job/assistant.dm [hunk 1]
```diff
@@ -28,6 +28,10 @@
 	H.equip_or_collect(new /obj/item/clothing/under/color/grey(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
 	H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
+	if(prob(1))
+		H.equip_or_collect(new /obj/item/weapon/storage/bag/plasticbag(H), slot_head)
```
Comment (SonixApache) [inline code/game/jobs/job/assistant.dm]: are you for real

Comment (jknpj) [reply code/game/jobs/job/assistant.dm]: it's a feature

File: code/game/jobs/job/civilian.dm [hunk 2]
```diff
@@ -472,7 +477,9 @@
 		H.equip_or_collect(new /obj/item/clothing/under/suit_jacket/red(H), slot_w_uniform)
 		//H.equip_or_collect(new /obj/item/device/pda/librarian(H), slot_belt)
 		H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
-		H.put_in_hands(new /obj/item/weapon/barcodescanner(H))
+		var/obj/item/weapon/storage/bag/plasticbag/shitterbag = new /obj/item/weapon/storage/bag/plasticbag(H)
```
Comment (ghost) [inline code/game/jobs/job/civilian.dm]: >shitterbag

File: code/game/jobs/job/assistant.dm (hunk 1 above)
Comment (yclatious) [inline code/game/jobs/job/assistant.dm]: My sides

Comment (Kurfursten) [reply code/game/jobs/job/assistant.dm]: While funny, I don't think we should do this for the designated new player job. People might suffocate before learning the controls.

**Comment by Ephx1** [comment]
Cool, but when do you make them look not shit?

**Comment by jknpj** [comment]
@Ephx1 when you sprite it

**Comment by Exxion** [comment]
Wow, service will get plastic bags now

File: code/game/jobs/job/assistant.dm [hunk 3]
```diff
@@ -28,6 +28,10 @@
 	H.equip_or_collect(new /obj/item/clothing/under/color/grey(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
 	H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
+	if(prob(1) && H.client.player_age >= 15)
+		H.equip_or_collect(new /obj/item/weapon/storage/bag/plasticbag(H), slot_head)
```
Comment (Exxion) [inline code/game/jobs/job/assistant.dm]: [Honk]

File: code/game/jobs/job/assistant.dm (hunk 1 above)
Comment (despotate) [reply code/game/jobs/job/assistant.dm]: If you can't learn the interface within one minute of joining you don't deserve to stay on this server

casuals please leave

File: code/game/jobs/job/assistant.dm (hunk 3 above)
Comment (gbasood) [reply code/game/jobs/job/assistant.dm]: remove

**Review by gbasood — CHANGES_REQUESTED** [review]
remove the RNG bag on head

File: code/game/jobs/job/assistant.dm [hunk 4]
```diff
@@ -28,7 +28,7 @@
 	H.equip_or_collect(new /obj/item/clothing/under/color/grey(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
 	H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
-	if(prob(1))
+	if(prob(1) && H.player_age >= 15)
```
Comment (SonixApache) [inline code/game/jobs/job/assistant.dm]: I guess that works. Sort of.

**Comment by Probe1** [comment]
```

	if(prob(1) && H.client.player_age >= 15)
 		H.equip_or_collect(new /obj/item/weapon/storage/bag/plasticbag(H), slot_head)
 	else
 		H.put_in_hands(new /obj/item/weapon/storage/bag/plasticbag(H))

```

👁‍🗨

**Review by gbasood — APPROVED** [review]

**Comment by gbasood** [comment]
Can you take a look at #13929 while you're at it

**Comment by Exxion** [comment]
Did you actually read the OP, Sood?

**Comment by jknpj** [comment]
@gbasood i fixed it, sorry for the trouble.

**Pull request merged**